lib/cocoapods/validator.rb

```diff
@@ -910,6 +924,17 @@ def podfile_from_spec(platform_name, deployment_target, use_frameworks = true, t
           else
             pod name, :podspec => podspec.to_s, :inhibit_warnings => false
           end
+
+          additional_path_pods.each do |podspec_path|
+            podspec_name = File.basename(podspec_path, '.*')
+            pod podspec_name, :path => File.dirname(podspec_path), :inhibit_warnings => false
```

Comment:
why would we not inhibit warnings? Those are external pods not currently being validated. I would expect to remove `:inhibit_warnings` here and below.

Reply:
good point

Reply:
Fixed